src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## Unreleased
 
+## 1.0.0-rc9.4 (source code moved to contrib repo)
```

Comment:
https://www.nuget.org/packages/OpenTelemetry.Instrumentation.StackExchangeRedis/1.0.0-rc9.4 was already done from main repo 😆

Reply:
Fixed this 😂.